Components/Pages/createGoal.js

```diff
@@ -55,6 +55,7 @@ const createGoal = ({ navigation, goToPassword, createNew }) => {
         placeholder="Title"
         keyboardType="default"
         autoCapitalize="words"
+        maxLength={12}
```

Comment:
@ChrisCruze I kinda felt like 12 was too short when I creating test goals. Not sure, lol
Also, should we autoCapitalize each word? I liked it when it was autoCapitalize first word. 
This is how they would look: 
- **Call My Mom** 
- **Call my mom**